backtesting/_plotting.py

```diff
@@ -607,9 +615,16 @@ def __eq__(self, other):
 
     custom_js_args = dict(ohlc_range=fig_ohlc.y_range,
                           source=source)
+    if plot_pl:
+        custom_js_args.update(pl_range=fig_pl.y_range)
```

Comment:
This change was not required. I think I might have added that while I was trying to see if I can add auto scale feature for P/L chart. Since it's already been added in the commit, I still see a value in keeping it for future use, But if you think that it's not worth adding it right now and will cause confusion then I can remove it. Let me know.